content/departments/cloud/technical-docs/oidc_site_admin.md

```diff
@@ -1,107 +1,7 @@
 # Using OpenID Connect for Site Admin Access
 
-# Purpose and Background
+Sourcegraph employees access to managed instances application interface (Web UI) is restricted to essential personnel only. This ensures Sourcegraph is able to help customers troubleshoot issues and deliver a smooth experience. We utilize OpenID Connect to enable Sourcegraph employees access to customer instance to make sure there is an audit trail for every access.
 
-[RFC 576: Allow disabling sign-up for OpenID Connect auth providers](https://docs.google.com/document/d/1Op6jum_SQJSU5KeJlJJEED1QYFdXOO30botb3BgEO6U/edit) provided the necessary changes to allow Sourcegraph to use their internal Okta Auth provider as a way to grant Site Admin access to managed instances. Okta provides 2FA enforcement and audit logs which help improve the security posture of Managed Instance access.
+# Implementation
 
-# Procedure
-
-These instructions are adapted from [Sourcegraph Docs Site - User authentication (SSO) instructions](https://docs.sourcegraph.com/admin/auth#openid-connect).
-
-## Adjusting License for increase seat count
```

Comment:
Is the adjusting of licenses no longer needed?

Reply:
I just moved it to a FAQ below